### PR TITLE
Inline virtualization

### PR DESCRIPTION
**Pull request by martingust**

### CODE REVIEW

**Comment by martingust** [comment]
When implemented this will get rid of the need of a surrounding fixed height container, the list can be inlined with the rest of the document also meaning there could be multiple `virtual-repeat`:s after each other. This will also use window scroll which will allow mobile browsers to go in full screen mode while scrolling and the scroll to top functionality when tapping the status bar on iOS devices should work as well.

In short with this in place all that is needed to get started using the `virtual-repeat` will be to install the plugin, replace `repeat.for` with `virtual-repeat.for` and make sure each row has equal height.  

More info https://github.com/aurelia/ui-virtualization/issues/29, thanks for ideas @YipYipX4!

**Comment by EisenbergEffect** [comment]
This is super exciting stuff! I can't wait :)
